pkg/mutate/mutate.go

```diff
@@ -79,6 +79,14 @@ func SetConfig(configFile any, configType types.MediaType) Mutation {
 	}
 }
 
+// SetConfigArtifactType replaces the artifact type with the specified value.
+func SetConfigArtifactType(artifactType string) Mutation {
```

Comment:
It's not the artifact type of the _config_ I don't think?

```suggestion
// SetArtifactType sets the artifact type to the specified value.
func SetArtifactType(artifactType string) Mutation {
```